hr_timesheet_sheet_attendance/tests/test_attendance_by_day.py

```diff
@@ -0,0 +1,73 @@
+from odoo.addons.hr_timesheet_sheet_attendance.tests.\
+    hr_timesheet_sheet_test_cases import HrTimesheetTestCases
+from odoo import fields
+
+
+class TestAttendanceByDay(HrTimesheetTestCases):
```

Comment:
Can you add a test case for attendances spanning on 2 days with a gap ?
e.g. : 
2019-04-02 19:00:00 to 2019-04-03 05:00:00
2019-04-03 19:00:00 to 2019-04-04 05:00:00